.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Mocha Tests",
+      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+      "args": ["src/execution/__tests__/executor-test.ts"],
+      "internalConsoleOptions": "openOnSessionStart",
+      "skipFiles": ["<node_internals>/**"]
+    }
+  ]
+}
```

Comment:
you might want to set up a global .gitignore file, I doubt we'll want to commit this

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c